src/inputs/radio-group/story-helper.ts

```diff
@@ -6,7 +6,7 @@ export const getRadioGroupArgTypes = (): ArgTypes => ({
     direction: {
         name: 'direction',
         type: { name: 'string' },
-        description: 'Radio buttons alignment direction',
+        description: 'Checkboxes alignment direction',
```

Comment:
```suggestion
        description: 'Radios alignment direction',
```

Reply:
OMG!! Thanks for pointing out the mistakes🫠